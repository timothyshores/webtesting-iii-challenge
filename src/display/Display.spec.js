import React from 'react';
import Display from './Display';

import {
    render,
    fireEvent,
    cleanup,
    waitForElement,
} from 'react-testing-library';

import renderer from 'react-test-renderer';

import 'jest-dom/extend-expect'

afterEach(cleanup)

describe('<Display />', () => {
    it('makes the snapshot', () => {
        expect(renderer.create(<Display />).toJSON()).toMatchSnapshot();
    });
    it('displays if gate is open/closed and if it is locked/unlocked', () => {

    });
    it("displays 'Closed' if the closed prop is true and 'Open' if otherwise", () => {

    });
    it("displays 'Locked' if the locked prop is true and 'Unlocked' if otherwise", () => {

    });
    it('uses the red-led class when locked or closed ', () => {

    });
    it('uses the use the green-led class when unlocked or open', () => {

    });

});