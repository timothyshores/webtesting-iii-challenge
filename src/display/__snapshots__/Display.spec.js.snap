// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> makes the snapshot 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;
